.circleci/config.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           - POSTGRES_DB: api_test
           - POSTGRES_PASSWORD: 12345
       - image: redis:6
+    parallelism: 4
```

Comment:
Does this also create 4 databases for each run?

Reply:
Yes. Basically it runs the same job 4 times with the full config and just split the tests using some circleci utils

Reply:
Perfect!